runtime/onert/backend/gpu_cl/open_cl/OpenclWrapper.cc

```diff
@@ -108,23 +108,33 @@ absl::Status LoadOpenCL()
 #endif
 }
 
+void UnloadOpenCL(void **libopencl)
+{
+  if (*libopencl)
+  {
+    dlclose(*libopencl);
```

Comment:
(I think I lost what I added here...anyway...)
as `_handle` is not updated, we can use `void UnloadOpenCL(void *libopencl)`
and this makes here without two `**` and caller without `&` which looks better to me.